libs/librepcb/common/dialogs/aboutdialog.h

```diff
@@ -0,0 +1,27 @@
+#ifndef ABOUTDIALOG_H
+#define ABOUTDIALOG_H
+
+#include <QDialog>
+#include <QAbstractButton>
+#include "../version.h"
+
+namespace Ui {
+class AboutDialog;
+}
+
+class AboutDialog : public QDialog
+{
+    Q_OBJECT
+
+public:
+    explicit AboutDialog(QWidget* parent, const librepcb::Version& appVersion, const QString& gitVersion, const QString& buildDate);
+    ~AboutDialog();
+
+private slots:
+    void on_buttonBox_clicked(QAbstractButton *button);
+
+private:
+    Ui::AboutDialog *ui;
+};
+
+#endif // ABOUTDIALOG_H
```

Comment:
For the whole header file: Please take a look at similar headers, for example `stroketextpropertiesdialog.h`.
- Missing license header
- Missing namespace
- Missing default/copy constructors and assignment operator
- Coding style in general (indentation, `noexcept` keyword, ...)
- Member `ui` could be a `QScopedPointer`

sorry ;)

Reply:
Yeah, I used the generator and did not *yet* add the license header etc. But it was 4:30 in the morning and I was tired, so I decided to go to sleep instead :stuck_out_tongue:

Reply:
Ah, I see :)

Reply:
Style guide should be applied now.